filament/backend/src/opengl/ShaderCompilerService.cpp

```diff
@@ -584,6 +584,9 @@ void ShaderCompilerService::compileShaders(OpenGLContext& context,
             outShaders[i] = shaderId;
         }
     }
+
+    // Hint to release the resources allocated by the shader compiler.
+    glReleaseShaderCompiler();
```

Comment:
What are you attempting to solve here? More shaders may need compilation, and this would create unnecessary churn.

Reply:
Okay, other shaders will be compiled.
Check the OpenGL specification.
This hint does not prevent later use of the shader compiler.
The idea is to deallocate specific resources from previous compilations, not common resources.
Obviously this will depend on the quality of the OpenGL implementation.

Reply:
I know it does not prevent future compilations, but is this actually fixing or improving anything?

Reply:
Yes, it is improving memory usage.

Reply:
I understand what it's *supposed* to do but have you measured its effect? We would also need to test this on all targets.

Reply:
I've only tested it on a few Filament samples on Linux.
I understand if you prefer not to proceed with the change due to the uncertainty of different OpenGL implementations on different devices.